modmesh/pylibmgr.py

```diff
@@ -27,10 +27,48 @@
 import sys
 import os
 import time
+import importlib.abc
+import importlib.machinery
 
 lib_path = {}
 
 
+class ModmeshPathFinder(importlib.abc.MetaPathFinder):
+    def find_spec(self, lib_name, path, target=None):
```

Comment:
@j8xixo12 I still don't get it.  How does this `ModmeshPathFinder` locate the directory `thirdparty/`?  It's not written anywhere.  I must miss something and please help me understand.